src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -81,117 +112,278 @@ public void focus() {
+            if (newValue != null) {
+                if ("Command Summary".equals(newValue)) {
+                    helpContentFlow.getChildren().clear(); //
+                    commandSummaryTable.setVisible(true);
+                } else {
+                    commandSummaryTable.setVisible(false);
+                    updateTextFlow(contentMap.get(newValue));
+                }
+            }
+        });
+    }
 
-            ### Marking a person as unpaid: `unpaid`
+    /**
+     * Initializes the command summary data for the TableView.
+     */
+    private void initializeCommandSummaryData() {
+        actionColumn.setCellValueFactory(cellData -> new javafx.beans.property.SimpleStringProperty(cellData.getValue()[0]));
+        formatColumn.setCellValueFactory(cellData -> new javafx.beans.property.SimpleStringProperty(cellData.getValue()[1]));
+        exampleColumn.setCellValueFactory(cellData -> new javafx.beans.property.SimpleStringProperty(cellData.getValue()[2]));
+
+        commandSummaryData.clear();
+
+        commandSummaryData.addAll(
+                new String[]{"Add", "add n/NAME p/PHONE_NUMBER\n e/EMAIL a/ADDRESS b/BIRTHDAY [t/TAG]…​", "add n/James Ho p/22224444 \ne/jamesho@example.com a/123, \nClementi Rd, 1234665 b/11 09 2001 \nt/friend t/colleague"},
+                new String[]{"Clear", "clear", ""},
+                new String[]{"Delete", "delete INDEX", "delete 3"},
+                new String[]{"Edit", "edit INDEX [n/NAME] [p/PHONE_NUMBER]\n [e/EMAIL] [a/ADDRESS] [t/TAG]…​", "edit 2 n/James Lee\n e/jameslee@example.com"},
+                new String[]{"Find", "find KEYWORD [MORE_KEYWORDS]", "find James Jake"},
+                new String[]{"List", "list", ""},
+                new String[]{"Help", "help", ""},
+                new String[]{"Paid", "paid INDEX", "paid 3"},
+                new String[]{"Unpaid", "unpaid INDEX", "unpaid 3"}
+        );
+        commandSummaryData.removeIf(row -> row == null || row.length == 0 ||
+                (row[0].isEmpty() && row[1].isEmpty() && row[2].isEmpty()));
+
+        commandSummaryTable.setItems(commandSummaryData);
+
+        commandSummaryTable.setColumnResizePolicy(TableView.CONSTRAINED_RESIZE_POLICY);
+    }
 
-            Marks the specified person from the FART book as unpaid.
+    /**
+     * Updates the TextFlow with bold, large titles, regular body text, hyperlinks, and code formatting for text in backticks.
+     */
+    private void updateTextFlow(String content) {
+        helpContentFlow.getChildren().clear();
 
-            ### Exiting the program : `exit`
+        String[] parts = content.split("\n", 2);
+        String titleText = parts[0];
+        String bodyText = parts.length > 1 ? parts[1] : "";
 
-            Exits the program.
+        Text title = new Text(titleText + "\n");
+        title.setStyle("-fx-font-weight: bold; -fx-font-size: 18px;");
```

Comment:
CMIIW, but I think you can abstract the styles out into the css file or the fxml file to keep the java code clean